graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/DgsSchemaProviderTest.kt

```diff
@@ -86,8 +86,9 @@ internal class DgsSchemaProviderTest {
                 applicationContextMock,
                 Optional.empty(),
                 Optional.empty(),
-                Optional.empty()
-            ).findSchemaFiles("notexists")
+                Optional.empty(),
+                listOf("classpath*:notexists/**/*.graphql*")
```

Comment:
Should we add coverage for the following variants?
* classpath with files found
* multiple locations with files found

Reply:
I will have a look tomorrow.

Reply:
Tests for multiple files and multiple locations are now added.